app/brave_generated_resources.grd

```diff
@@ -196,6 +196,12 @@
       <message name="IDS_WIDEVINE_DONT_ASK_AGAIN_CHECKBOX" desc="Checkbox for prevent showing widevine install prompt">
         Don't ask again
       </message>
+      <message name="IDS_SETTINGS_ENABLE_WIDEVINE_TITLE" desc="Text fragment for enabling widevine component. 'Widevine' is the name of a plugin and should not be translated.">
+        Enable Widevine
+      </message>
+      <message name="IDS_SETTINGS_ENABLE_WIDEVINE_DESC" desc="Text fragment for enabling widevine component. 'Widevine' is the name of a plugin and should not be translated.">
+        Uses Widevine digital rights management (DRM) component to view streaming video and other content in the browser.
```

Comment:
Maybe change to `Use Widevine digital rights management (DRM) component to view protected streaming video and other content in the browser.` (fixed `Uses` => `Use` and added `protected` in there)

Reply:
Chagned.